test/test-init-config.ts

```diff
@@ -0,0 +1,336 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as assert from 'assert';
+import * as nock from 'nock';
+import * as request from 'request';
+import * as sinon from 'sinon';
+
+import {initConfig} from '../src/index';
+
+const gcpMetadata = require('gcp-metadata');
+
+
+nock.disableNetConnect();
+const metadataAPI = 'http://metadata.google.internal/computeMetadata/v1';
```

Comment:
This is unused. Did you run lint?

Reply:
I've run gts check and gts fix -- am I missing something?

Reply:
Hmm.. that should have caught it.. if not, that sounds like a bug in gts.